api/routers.py

```
@@ -2,13 +1,30 @@
-    """لایونس‌چک ساده: 200 و {"status": "ok"}؛ فقط GET/HEAD و کش غیرفعال."""
+    """سلامت سرویس: JSON {"status": "ok"} با HTTP 200. بدون کش."""
     return JsonResponse({"status": "ok"})
 
-urlpatterns = [path('health/', health, name='health')]
+# Create version views
+version_list = VersionViewSet.as_view({'get': 'list'})
+version_revert = VersionViewSet.as_view({'post': 'revert'})
+
+urlpatterns = [
+    path('health/', health, name='health'),
+    path('versions/<str:resource_type>/<uuid:resource_id>/', version_list, name='version-list'),
```

Ruff (0.12.2)

Line too long (96 > 88)

(E501)